common/core/desktop/doc/BUILDING.md

```diff
@@ -90,16 +167,34 @@ Place this in the same folder as you extracted kmcomp.exe, and
 
 ### macOS
 
-You can get the official Python installer from the official Python site:
-<https://www.python.org/downloads/mac-osx/>
+* Install Python 3
 
-Install meson and rust:
+  You can get the official Python installer from the official Python site:
+  <https://www.python.org/downloads/mac-osx/>
 
-```bash
-brew install meson    # if you haven't already installed via pip
-curl https://sh.rustup.rs -sSf | sh
-rustup target add i686-pc-windows-msvc
-```
+* Install meson:
+
+  ```bash
+  brew install meson    # if you haven't already installed via pip
+  ```
+
+* Install Rust:
+
+  ```bash
+  curl https://sh.rustup.rs -sSf | sh
+  rustup target add wasm32-unknown-unknown
+  ```
+
+* Install wasm-pack:
+
+  ```bash
+  curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+  ```
+
+* Install Enscripten (including environment update):
+  <https://emscripten.org/docs/getting_started/downloads.html#sdk-download-and-install>
+
```

Comment:
wasm-pack (0.9.1, depends on brew's rust) and emscripten (2.0.22), and rust itself for that matter are available via brew.

Reply:
`brew install emscripten wasm-pack rustup-init meson python3  && rustup target add wasm32-unknown-unknown` ? plus emscripten environment update

Reply:
I'm going to setup a macOS VM to test this option -- my local dev machine is already corrupted by other ways of installing... 😆

Reply:
The mac build environment work is ongoing and will appear in a separate PR